dbconnection.php

```diff
@@ -6,10 +6,11 @@
 DEFINE ('DB_NAME', 'Store');
 
 // Make the connection:
-$conn = @mysqli_connect (DB_HOST, DB_USER, DB_PASSWORD, DB_NAME) OR die ('Could not connect to MySQL: ' . mysqli_connect_error() );
+$mysqli = new mysqli("localhost", "root", "", "Store");
 
-if ($conn->connect_error) {
-    die("Connection failed: " . $conn->connect_error);
-} 
+if ($mysqli->connect_errno) {
+    printf("Connect failed: %s\n", $mysqli->connect_error);
+    exit();
```

Comment:
I took entire DB connection prom MySQLi DB connection manual. I assume PHP Manual contributors know their stuff better than us. Let's not pick up on small things.

Reply:
Just making sure that before it is merged to master that it works. I merged it and it broke a lot on my machine.

Reply:
Gotta figure it out still. DB connection isn’t working anymore. I pulled everything and down and just intially started up my local host. No biggie. I’ll figure it out it may just be dB crendtials are not the same.

Reply:
Probably. Or your WAMP doesn't support MySQLi? Which doesn't make sense. PHP errorcode on connection can help. Good Luck!